frontend/src/employee-frontend/components/unit/unit-reservations/ChildReservationsTable.tsx

```diff
@@ -150,9 +158,32 @@ export default React.memo(function ChildReservationsTable({
               selectedGroup,
               shiftCareAllowedForChild
             )
-          }).length
-      ),
-    [days, selectedGroup, childBasics]
+          })
+          .map((child) => child.childId)
+        const occupancy = childBasics.reduce((occupancy, child) => {
```

Comment:
This calculation could be done in the backend. Also family daycare (perhepäivähoito) has different calculation rules.

Reply:
> family daycare (perhepäivähoito) has different calculation rules

Yeah, forgot that.

Moving calculation to backend needs major changes, because currently a lot is happening in frontend.

Reply:
Fixed!